ci/lib/install_functions.inc.sh

```diff
@@ -546,19 +555,34 @@ msys_install() {
     make
     pkg-config
     mingw64/mingw-w64-x86_64-cmake
-    mingw64/mingw-w64-x86_64-gcc
-    mingw64/mingw-w64-x86_64-json-c
     mingw64/mingw-w64-x86_64-python3
   )
 
+  if [ "${CC}" = "gcc" ]; then
+    packages+=(mingw64/mingw-w64-x86_64-gcc 
+               mingw64/mingw-w64-x86_64-json-c
+    )
+  else
+   packages+=(clang64/mingw-w64-clang-x86_64-clang 
+              clang64/mingw-w64-clang-x86_64-openmp
+              clang64/mingw-w64-clang-x86_64-libc++
+              clang64/mingw-w64-clang-x86_64-libbotan
+              clang64/mingw-w64-clang-x86_64-libssp
+              clang64/mingw-w64-clang-x86_64-json-c
+              clang64/mingw-w64-clang-x86_64-libsystre
+   ) 
```

Comment:
These lines need to indent by one more space.

Reply:
Fixed.